### PR TITLE
Data structure imports

### PR DESCRIPTION
**Pull request by mudaliarkartik**
#21

### CODE REVIEW

**Review by mnil2 — CHANGES_REQUESTED** [review]
You've put a big dent in the import failures, but there are still 3 according to travis.

graph.floyd_warshall, graph.dijkstra and graph.bellman_ford all try to read input.  Check the raw log of the travis build for occurances of `test_import_module[thealgorithms.data_structures`

File: thealgorithms/data_structures/stacks/__init__.py [hunk 1]
```diff
@@ -1,23 +0,0 @@
-class Stack:
```
Comment (slarse) [inline thealgorithms/data_structures/stacks/__init__.py:1]: Hmm, not sure about outright removing this stack. Maybe we should add it in a `simple_stack.py` module, or something.

**Review by slarse — CHANGES_REQUESTED** [review]

File: thealgorithms/data_structures/stacks/__init__.py (hunk 1 above)
Comment (mudaliarkartik) [reply thealgorithms/data_structures/stacks/__init__.py:1]: This part of code is not being used any where

Comment (slarse) [reply thealgorithms/data_structures/stacks/__init__.py:1]: Most code is not being used anywhere, it's a library.

Comment (mudaliarkartik) [reply thealgorithms/data_structures/stacks/__init__.py:1]: Added the file as `simple_stack.py`

**Review by mnil2 — APPROVED** [review]

**Comment by mnil2** [comment]
Looks good to me now

**Review by slarse — APPROVED** [review]
LGTM

**Pull request merged**